src/build.netcore.props

```diff
@@ -3,7 +3,8 @@
   <Import Project="$(MSBuildThisFileDirectory)build.common.props" />
 
   <PropertyGroup>
-    <NetCoreVersion>netcoreapp3.1</NetCoreVersion>
```

Comment:
>netcoreapp3.1 [](http://example.com/codeflow?start=20&length=13)

do we want to keep .net core 3.1?
[https://devblogs.microsoft.com/dotnet/net-core-3-1-will-reach-end-of-support-on-december-13-2022/](https://devblogs.microsoft.com/dotnet/net-core-3-1-will-reach-end-of-support-on-december-13-2022/)

Reply:
I think we need to keep all existing and add 6.0